README.md

```diff
@@ -1,8 +1,8 @@
 # Gradle Plugin for Datadog Android SDK
 
-> This plugin is used to upload your Proguard/Dexguard/R8 mapping files to Datadog to get a complete RUM Error Tracking experience.
+> This plugin is used to upload your Proguard/Dexguard/R8 mapping files and NDK symbol files to Datadog to get a complete RUM Error Tracking experience.
 
-Once your mapping file are uploaded, any error/crash reported through RUM will have a deobfuscated stacktrace, enabling an easy diagnosis of the root cause of the error.
+Once your mapping file are uploaded, any error/crash reported through RUM will have a deobfuscated and symbolicated stacktrace, enabling an easy diagnosis of the root cause of the error.
```

Comment:
minor: maybe use `/` here? because stacktrace is either obfuscated or non-symbolicated.

```suggestion
Once your mapping file are uploaded, any error/crash reported through RUM will have a deobfuscated/symbolicated stacktrace, enabling an easy diagnosis of the root cause of the error.
```